ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -197,6 +206,8 @@ def install_lso_cluster(self):
 
         logger.info("Confirm new storage class")
         self.do_click(self.dep_loc["yes"], enable_screenshot=True)
+        if config.ENV_DATA.get("mcg_only_deployment", False):
+            return
```

Comment:
I didn't understand this line, could you please explain?

Reply:
This is a way to exit a function (which has no return value).
If we work with MCG_ONLY mode

Reply:
But I think the default value `False` could be omitted, isn't it?